tests/yast2_gui/yast2_users.pm

```diff
@@ -16,11 +16,107 @@ use strict;
 use warnings;
 use testapi;
 
+use utils qw(type_string_slow_extended);
```

Comment:
It might happen that static code checks in travis will report this in the future. Better to use ```use utils 'type_string_slow_extended';``` than to have an array of one item. But this is neat picking stuff :D